### ISSUE TITLE
Linter XO ignores disable directives

### ISSUE THREAD

**Issue by shanoaice** [xojs/vscode-linter-xo#72]
When I tried to disable check for a line like this:
``` js
'code with problem' // eslint-disable-line
```
The extension ignores the disable directive and continued to report the error

**Comment by sindresorhus** [comment]
You have to specify which rule(s) to disable.

**Comment by noctumsempra** [comment]
> You have to specify which rule(s) to disable.

Nop. It should just NOT check that line.

**Comment by papb** [comment]
Are you sure the linter is not reporting an abusive disable? XO has a rule that disallows using `eslint-plugin-disable`

**Comment by noctumsempra** [comment]
The few times I've put that special comment in Webstorm to bypass some circumstantial checking, this IDE took it correctly. Given eslint directives are made to be used if needed, any JS editor/IDE should support it, imho.

**Comment by papb** [comment]
@noctumsempra Yeah, I know, I was just asking to be sure if the directive wasn't working, or the directive did work as it should and XO is now reporting another error (i.e. the fact that you're using `eslint-disable-line`, which by itself is an error)

**Comment by Spence-S** [comment]
stale and likely [xo](https://github.com/xojs/xo) issue that has been resolved by now
